google_cloudsql_mysql/README.md

```diff
@@ -132,12 +132,19 @@ output "mysql_database" {
 | <a name="input_maintenance_window_update_track"></a> [maintenance\_window\_update\_track](#input\_maintenance\_window\_update\_track) | n/a | `string` | `"stable"` | no |
 | <a name="input_network"></a> [network](#input\_network) | Network where the private peering should attach. | `string` | `"default"` | no |
 | <a name="input_project_id"></a> [project\_id](#input\_project\_id) | n/a | `string` | `null` | no |
+| <a name="input_query_insights_enabled"></a> [query\_insights\_enabled](#input\_query\_insights\_enabled) | See: https://cloud.google.com/sql/docs/postgres/using-query-insights | `bool` | `false` | no |
```

Comment:
should this be a link to the MySQL version?

Reply:
absolutely, thanks, fixed it :+1: